BackEnd/src/main/java/com/exadel/carpoolfree/controller/ChatController.java

```diff
@@ -1,84 +1,27 @@
 package com.exadel.carpoolfree.controller;
 
-import com.exadel.carpoolfree.model.Message;
-import com.exadel.carpoolfree.repository.MessageRepository;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.messaging.handler.annotation.DestinationVariable;
+import com.exadel.carpoolfree.service.ChatService;
 import org.springframework.messaging.handler.annotation.MessageMapping;
-import org.springframework.messaging.handler.annotation.Payload;
-import org.springframework.messaging.handler.annotation.SendTo;
-import org.springframework.messaging.simp.SimpMessageHeaderAccessor;
-import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import java.util.Map;
 
-//@Controller
-//public class ChatController {
-//
-//    @MessageMapping("/chat.sendMessage")
-//    @SendTo("/topic/public")
-//    public Message sendMessage(@Payload Message message) {
-//        return message;
-//    }
-//
-//    @MessageMapping("/chat.addUser")
-//    @SendTo("/topic/driveId/{driveId}")
-//    public Message addUser(@Payload Message message,
-//                           SimpMessageHeaderAccessor headerAccessor) {
-//        // Add username in web socket session
-//        headerAccessor.getSessionAttributes().put("username", message.getUser().getFirstName());
-//        return message;
-//    }
-//
-//}
 
 @RestController
 @RequestMapping(value = "/api/socket")
 @CrossOrigin("*")
 public class ChatController {
 
-    private final MessageRepository messageRepository;
+    private final ChatService chatService;
 
-    public ChatController(MessageRepository messageRepository) {
-        this.messageRepository = messageRepository;
+    public ChatController(ChatService chatService)
+    {
+        this.chatService = chatService;
     }
 
-    @Autowired
-    private SimpMessagingTemplate simpMessagingTemplate;
-
     @MessageMapping("/send/message")
-    public Message sendMessage(@Payload Message message) {
-        messageRepository.save(message);
-        this.simpMessagingTemplate.convertAndSend("/socket-publisher/" + message.getDriveId() + "_" + message.getUser().getId(), message);
-        return message;
-    }
-
-
-   /* @MessageMapping("/send/message")
```

Comment:
If we need something back, we have history in GIT anyway. We don't think our current colution is incorrent, do we?